src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -2750,10 +2750,29 @@ public final class JavadocTokenTypes {
             + RULE_TYPES_OFFSET;
 
     /**
-     * Non-special empty html tag.
+     * Non-special empty HTML tag.
+     *
+     * <p>Example:</p>
+     * <pre>
+     * &#47;**
+     *  * Example for empty tag.
+     *  *
+     *  * &lt;custom /&gt;
+     *  *
+     *  * public class Test { }
```

Comment:
only `* &lt;custom /&gt;` should be there.

and please make tag to be named `justsometag` to make it clear it is for non html tags.